backend/challenges/RenameColumnAlternative/challenge.py

```diff
@@ -0,0 +1,27 @@
+import hypothesis.strategies as st
+from hypothesis.extra.pandas import data_frames, column, range_indexes
+import numpy as np
+import pandas as pd
+import pandas.testing as tm
+
+import Challenge from ??
+
+
+def transform(df: pd.DataFrame) -> pd.DataFrame:
+    df = df.rename(columns={"Value": "NewValue"})
+    return df
+
+# Since the "run" "initial" and "test" use cases all rely on the same param generators it makes sense to share them
+# Possibly as a dict 
+params = {
```

Comment:
Why do you put the df-creation function in a dict? It should work straight forward without that, e.g.:
`create_df = data_frames(...)`
And then just use the function call `create_df().example()` for generation, or 
`@given(df=create_df`)`